### DISCUSSION TITLE
Budget Impact Model in R

### DISCUSSION THREAD

**Discussion by RayHendy** [HTAhackathon2024/.github#11] [Ideas]
### What is the problem we're trying to solve

Write code in R for budget impact model (BIM)

### How will we tackle the problem?

- Pick disease area
- Get data
- Implement code

### Which tools will we use?

`utils::install.packages("devtools")`

`devtools::install_github("ready4-dev/bimp")`

### Previous work

Here's a link to a previous BIM (not mine!) https://github.com/ready4-dev/bimp

**Comment by n8thangreen** [comment]
`bimp` doesnt seem to have any vignettes/examples. Perhaps writing some using this would be useful?

**Comment by RayHendy** [comment]
@n8thangreen chatting with a colleague at work and they would prefer I try and code some survival analysis instead. Should I delete this idea?

**Reply by n8thangreen** [reply]
Up to you Raymond!

**Comment by aponzo125** [comment]
I would be interested in working on this !

**Reply by tkamiya-hiqa** [reply]
I am interested in BIAs using R as well.

I wonder if the crux of the modelling challenge is system-specific though.
